src/libs/API.js

```diff
@@ -237,7 +237,6 @@ function Authenticate(parameters) {
         partnerUserSecret: parameters.partnerUserSecret,
         twoFactorAuthCode: parameters.twoFactorAuthCode,
         doNotRetry: true,
-        isViaExpensifyCash: true,
```

Comment:
I'm not sure we wanted to remove this line

Reply:
ah crap looks like you're right. Why didn't we change that one in api.php as well?

Reply:
Why not? I'm not seeing it...

Reply:
We forgot to update the command in api.php to look for the referer, so it was possible to authenticate through e.cash without a GH token. 
CP'd this fix https://github.com/Expensify/Web-Expensify/pull/30071

Reply:
Ah got it. I was looking at `master` wondering what the problem was but you've fixed it 😅